### PR TITLE
Add Template Toolkit language and icon

### PR DESCRIPTION
**Pull request by heinrich-S**
_**Fixes #IssueNumber**_

**Changes proposed:**

- [x] Add
Add [Template Toolkit](http://www.template-toolkit.org/) language and icon

### CODE REVIEW

**Comment by JimiC** [comment]
@heinrich-S Which is the `vscode` extension that supports this file extension?

**Comment by heinrich-S** [comment]
The vscode extension for TemplateToolkit is https://github.com/samosad/vscode-template-toolkit
I've removed the stroke-width, it's okay like this?

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -2546,6 +2546,12 @@ export const extensions: IFileCollection = {
       languages: [languages.t4],
       format: FileFormat.svg,
     },
+    {
+      icon: 'tt',
+      extensions: ['tt2', 'tt3'],
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: `tt3` is already supported by the `languadeId` (https://github.com/samosad/vscode-template-toolkit/blob/master/package.json#L41)

File: src/icon-manifest/languages.ts [hunk 2]
```diff
@@ -182,6 +182,7 @@ export const languages: ILanguageCollection = {
   swig: { ids: 'swig', defaultExtension: 'swig' },
   systemverilog: { ids: 'systemverilog', defaultExtension: 'sv' },
   t4: { ids: 't4', defaultExtension: 'tt' },
+  templatetoolkit: { ids: 'tt', defaultExtension: 'tt' },
```
Comment (JimiC) [inline src/icon-manifest/languages.ts]: In order to avoid overriding the `t4` file extension, change the `defaultExtension` for `tt` to `tt3`.

**Comment by JimiC** [comment]
`strokes` should not be removed but converted to `paths`.

File: src/icon-manifest/supportedExtensions.ts [hunk 3]
```diff
@@ -2548,7 +2548,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'tt',
-      extensions: ['tt2', 'tt3'],
+      extensions: ['tt2','tt'],
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: `tt` is also supported from the `languageId`. Please remove it entirely.

**Comment by heinrich-S** [comment]
I don't know much about svg, so I'm not realy understanding your request about `strokes` and `paths` . Can you point me to a sample file?

**Comment by JimiC** [comment]
@heinrich-S I'll just fix it for you. No worries.

**Pull request merged**